src/shared_modules/content_manager/src/components/componentsHelper.hpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Wazuh Content Manager
+ * Copyright (C) 2015, Wazuh Inc.
+ * Nov 29, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _COMPONENTS_HELPER_HPP
+#define _COMPONENTS_HELPER_HPP
+
+#include "json.hpp"
+#include "updaterContext.hpp"
+#include <map>
+#include <stdexcept>
+#include <string>
+
+namespace Components
+{
+    /**
+     * @brief Possible components' status.
+     *
+     */
+    enum Status
```

Comment:
Improve: I think you can use the `enum class` here instead of `enum`

Reply:
Didn't know that. Done!